src/chat/chat.gateway.ts

```
@@ -5,13 +5,14 @@ import {
   WebSocketGateway,
 } from '@nestjs/websockets';
 import { Socket } from 'socket.io';
+import { ChatMessageDto } from './dto/chat-message.dto';
 
 @WebSocketGateway()
 export class ChatGateway {
   @SubscribeMessage('message')
   handleMessage(
     @ConnectedSocket() client: Socket,
-    @MessageBody() payload: { message: string },
+    @MessageBody() message: ChatMessageDto,
```

ESLint

[error] 'message' is defined but never used.

(@typescript-eslint/no-unused-vars)